source/amazon/installation.rst

```diff
@@ -155,78 +160,45 @@ Plugin configuration
 
     # vi /var/ossec/etc/ossec.conf
 
-2. Add the following block of configuration to enable the integration, enter the AWS IAM User credentials you created before:
+2. Add the following block of configuration to enable the integration, enter the AWS IAM User credentials you created before and the AWS Account ID of the CloudTrail logs to be processed:
 
 .. code-block:: xml
 
+
     <wodle name="aws-cloudtrail">
       <disabled>no</disabled>
-      <bucket>wazuh-cloudtrail</bucket>
-      <access_key>insert_access_key</access_key>
-      <secret_key>insert_secret_key</secret_key>
-      <remove_from_bucket>no</remove_from_bucket>
       <interval>10m</interval>
       <run_on_start>no</run_on_start>
+      <error_not_skip><error_not_skip>
```

Comment:
This example is wrong: the correct value is `skip_on_error`.

Reply:
Good catch.  This was the original wording of the option.  I will fix...